src/embed/types.rs

```diff
@@ -184,6 +184,18 @@ pub struct PutRequest {
 #[derive(DeJson, SerJson)]
 pub struct PutResponse {}
 
+#[derive(DeJson)]
+pub struct DelRequest {
+    #[nserde(rename = "transactionId")]
+    pub transaction_id: u32,
+    pub key: String,
+}
+
+#[derive(DeJson, SerJson)]
```

Comment:
"ok" makes it sound like if ok is false there was a problem. how about 'was_present' or 'had' similar? at the very least we should have a comment.

Reply:
Yeah it needs to be `ok` for compat with the js, but I can use rename so at least the field is reasonable.

Reply:
Fixed